frontend/src/lib/utils.tsx

```diff
@@ -708,177 +708,178 @@ export function determineDifferenceType(
     }
 }
 
-const DATE_FORMAT = 'D MMM YYYY'
+const DATE_FORMAT = 'MMMM D, YYYY'
+const DATE_FORMAT_WITHOUT_YEAR = 'MMMM D'
```

Comment:
Where does this change the dates? Just the date picker, or other random places in the app?

Reply:
I checked. Just in the Date Filter for now, unless you use `dateFilterToText` with `isFormatted=true`, which is not used anywhere else.